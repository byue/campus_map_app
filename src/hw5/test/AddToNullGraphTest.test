# Tests that a graph can be created, and proper output
# for adding to a null graph.

# Create a graph
CreateGraph graph1

# Add node to a null graph
AddNode graph2 n1