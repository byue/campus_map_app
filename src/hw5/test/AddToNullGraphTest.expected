# Tests that a graph can be created, and proper output
# for adding to a null graph.

# Create a graph
created graph graph1

# Add node to a null graph
graph2 has not been created